src/components/CVE-2021-41773/CVE-2021-41773.tsx

```diff
@@ -63,40 +151,88 @@ export function CVE202141773() {
             "-w",
             values.version,
         ];
-        const output = await CommandHelper.runCommand("python3", args);
 
-        setOutput(output);
-        setLoading(false);
+        // Execute the CVE 2021-41773 command via helper method and handle its output or potential errors.
+        CommandHelper.runCommandWithPkexec("python3", args, handleProcessData, handleProcessTermination)
```

Comment:
Please confirm that you need root access to run this command. Otherwise change to runCommandGetPidAndOutput.